go.mod

```diff
@@ -2,10 +2,4 @@ module github.com/aws/aws-lambda-go
 
 go 1.18
 
-require github.com/stretchr/testify v1.6.1
-
-require (
```

Comment:
Per failing test workflows, I believe you forgot to `git add go.mod` after `go mod tidy`

Reply:
That's right Bryan. Closing this as it has been fixed by another PR (https://github.com/aws/aws-lambda-go/pull/446)